influxdb_iox/src/commands/sql/repl.rs

```diff
@@ -398,6 +400,7 @@ async fn scrape_query(
         .perform_query(ReadInfo {
             namespace_name: db_name.to_string(),
             sql_query: query.to_string(),
+            query_type: read_info::QueryType::Sql as i32,
```

Comment:
It's unfortunate that enumerated fields are not strongly typed on the generated types 😭

Reply:
I think a more idomatic way to do this is

```suggestion
            query_type: read_info::QueryType::Sql.into(),
```

So at least you don't have `as` sprinkled around

Reply:
Thanks! I thought I had to implement one of the Into / From traits to do that

Reply:
Updated all references 👍🏻